src/backend/base/langflow/services/database/models/api_key/model.py

```diff
@@ -13,6 +14,10 @@ def utc_now():
     return datetime.now(timezone.utc)
 
 
+def expire_time():
+    return utc_now() + timedelta(days=API_KEY_EXPIRATION_DAYS)
```

Comment:
I would move this function to an utils file

Reply:
there is a utc_now() function above it, should we move them together ?